### PR TITLE
Update readme files for some modules

### PR DESCRIPTION
**Pull request by weijiekoh**
Addresses #226

### CODE REVIEW

File: crypto/README.md [hunk 1]
```diff
@@ -49,3 +84,31 @@ Produces a signature of the given message using the private key.
...
+**`hash3 = (elements: Plaintext): BigInt`**: the Poseidon hash function for 3
+inputs.
+
+**`hash4 = (elements: Plaintext): BigInt`**: the Poseidon hash function for 4
+inputs.
+
+**`hash5 = (elements: Plaintext): BigInt`**: the Poseidon hash function for 5
+inputs.
+
+**`hash12 = (elements: Plaintext): BigInt`**: the Poseidon hash function for 12
```
Comment (corydickson) [inline crypto/README.md:112]: Does this implicitly use hashLeftRight by first using hash5 then hash6 on the remaining inputs? If so we may want to include this here so users are aware of the ordering.

**Review by corydickson — CHANGES_REQUESTED** [review]
Looking good, just a couple small things

File: core/README.md [hunk 2]
```diff
@@ -10,303 +10,89 @@ transform said data. This makes it easier to reason about the system, write
...
-#### **`copy`**
-
-Function signature:
-
-```ts
-(): Poll
-```
-
-Deep-copies and returns this object.
+To this end, we this submodule exposes a `MaciState` class and a `Poll` class.
```
[the remark this answers is not in the document]
Comment (weijiekoh) [reply core/README.md]: Fixed

File: crypto/README.md (hunk 1 above)
Comment (weijiekoh) [reply crypto/README.md:112]: Thanks for the suggestion. I've added the pseudocode in a new commit.

**Review by corydickson — APPROVED** [review]

**Pull request merged**